clients/hadoopfs/pom.xml

```diff
@@ -46,20 +46,20 @@
                                     </excludes>
                                 </filter>
                             </filters>
-			    <relocations>
```

Comment:
I think we agreed no tabs,evidently I only configured it for Java not XML.  Sorry ;-(.

Reply:
right, fixing - I gave the idea a chance in reformat